.cicd/build.sh

```diff
@@ -5,21 +5,14 @@ mkdir -p $BUILD_DIR
 CMAKE_EXTRAS="-DCMAKE_BUILD_TYPE='Release' -DENABLE_MULTIVERSION_PROTOCOL_TEST=true -DBUILD_MONGO_DB_PLUGIN=true"
 if [[ "$(uname)" == 'Darwin' ]]; then
     # You can't use chained commands in execute
-    if [[ "$GITHUB_ACTIONS" == 'true' ]]; then
+    if [[ $BUILDKITE == true ]]; then
+        CMAKE_EXTRAS="$CMAKE_EXTRAS -DBUILD_MONGO_DB_PLUGIN=true"
```

Comment:
The MongoDB flag is already enabled by default on line 5. This line can be removed entirely since the MongoDB tests pass in Actions.